src/PostgREST/QueryBuilder.hs

```diff
@@ -174,6 +174,6 @@ limitedQuery :: H.Snippet -> Maybe Integer -> H.Snippet
 limitedQuery query maxRows = query <> H.sql (maybe mempty (\x -> " LIMIT " <> BS.pack (show x)) maxRows)
 
 -- | Do a pg set_config(setting, value, true) call. This is equivalent to a SET LOCAL.
-setConfigLocal :: Text -> (Text, Text) -> H.Snippet
+setConfigLocal :: ByteString -> (ByteString, ByteString) -> H.Snippet
```

Comment:
Changed this signature to use ByteString because `JSON.encode` returned a ByteString. Using `encodeUtf8` on the big json spec to convert it to Text would have been wasteful.